### PR TITLE
Added Initial Time Packet on Login

### PR DESCRIPTION
**Pull request by OfficialKris**
## Description
Fixes a bug where when a player logs in, the time is shown to be incorrect for one second.

## Testing
- Logged in
- Set time to 18000
- Logout
- Login and notice time is always correct 

## Checklist
Things need to be done before this Pull Request can be merged.

- [x] Code is well-formatted and adheres to project style guidelines: `cargo fmt`
- [x] Code does not produce any clippy warnings: `cargo clippy`
- [x] All unit tests pass: `cargo test`
- [ ] I added new unit tests, so other people don't accidentally break my code by changing other parts of the codebase. [How?](https://doc.rust-lang.org/book/ch11-01-writing-tests.html)

### CODE REVIEW

**Comment by Snowiiii** [comment]
For some reason this does not work. If i set the time to 0 and rejoin it shows me the a different time

**Comment by OfficialKris** [comment]
I cannot reproduce, are there any additional steps you took?

File: pumpkin/src/world/mod.rs [hunk 1]
```diff
@@ -404,6 +404,9 @@ impl World {
         // Spawn in initial chunks
         player_chunker::player_join(self, player.clone()).await;
 
+        // Sends initial time
+        player.send_time(self).await;
```
Comment (Snowiiii) [inline pumpkin/src/world/mod.rs:408]: Also i think we should send this before sending chunks, like vanilla does it

**Comment by Snowiiii** [comment]
> I cannot reproduce, are there any additional steps you took?

mhh, nope

File: pumpkin/src/entity/player.rs [hunk 2]
```diff
@@ -396,6 +399,18 @@ impl Player {
         self.permission_lvl
     }
 
+    // sends the world time to just the player
```
Comment (Snowiiii) [inline pumpkin/src/entity/player.rs:402]: Doc comments are written using ///. Normal comments are just //.

**Comment by Snowiiii** [comment]
Thanks @OfficialKris

**Pull request merged**